### PR TITLE
python-packages tornado: 4.1 -> 4.2.1

### PR DESCRIPTION
**Pull request by FRidh**

### CODE REVIEW

**Comment by FRidh** [comment]
> The log length has exceeded the limit of 4 Megabytes

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -15138,16 +15138,17 @@ let
...
     propagatedBuildInputs = with self; [ backports_ssl_match_hostname_3_4_0_2 certifi ];
 
     src = pkgs.fetchurl {
       url = "https://pypi.python.org/packages/source/t/tornado/${name}.tar.gz";
-      sha256 = "0a12f00h277zbifibnj46wf14801f573irvf6hwkgja5vspd7awr";
+      sha256 = "a16fcdc4f76b184cb82f4f9eaeeacef6113b524b26a2cb331222e4a7fa6f2969";
     };
 
-    doCheck = false;
+    #doCheck = false;
```
Comment (lethalman) [inline pkgs/top-level/python-packages.nix]: Explain why

**Comment by FRidh** [comment]
Built and installed the package with Python 2.7 and 3.4. I didn't run `nox-review wip` because it would compile a lot.

@lethalman I removed `doCheck = false` because I couldn't find in neither the code nor in the log a reason for why it was set to false, and the package seems to build and install fine.

**Comment by lethalman** [comment]
Ops, thought it was `true`. Looks fine thanks.

**Pull request merged**